addons/mail/__manifest__.py

```diff
@@ -128,6 +128,7 @@
             'mail/static/src/**/primary_variables.scss',
```

Comment:
Commit message could mention that we add a hook method inside the "SampleServer" as well to be able to override the fields for which we want to return false for selections.

Currently it's very centered on the changes in mail.

The header could also mention the "web" module.